### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by lholmquist** [comment]
besides travis  :+1:  Jolly Good

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AeroGear Android Push [![Build Status](https://travis-ci.org/aerogear/aerogear-android-push.png)](https://travis-ci.org/aerogear/aerogear-android-push)
 
-AeroGear's Android libraries were built as jar, apklib and aar using [Maven](http://maven.apache.org/) and [android-maven-plugin](https://github.com/jayway/maven-android-plugin). The project follows the standard Android project layout as opposed to the standard Maven layout so sources will be in /src instead of /src/main/java and can be imported directly into IDE as an Android project.
+AeroGear's Android libraries were built as jar and aar packages using [Maven](http://maven.apache.org/) and the [android-maven-plugin](https://github.com/jayway/maven-android-plugin). The project follows the standard Maven layout so can be imported directly into most IDEs as a Maven project.
```
Comment (abstractj) [inline README.md]: I think this sentence is missing the "it" "The project follows the standard Maven layout so ***it*** can be imported directly"

File: README.md [hunk 2]
```diff
@@ -8,7 +8,12 @@ AeroGear for Android provides support for integrating with push. Currently only
 
 ## Building
 
-Please take a look of [step by step on our website](http://aerogear.org/docs/guides/aerogear-android/how-to-build-aerogear-android/)
+Please take a look of the [step by step guide](http://aerogear.org/docs/guides/aerogear-android/how-to-build-aerogear-android/) on our website.
```
Comment (abstractj) [inline README.md]: I think the correct would be "Please take a look ***at*** the....

**Comment by abstractj** [comment]
@danielpassos please apply the same fixes to:

https://github.com/aerogear/aerogear-android-core/pull/17
https://github.com/aerogear/aerogear-android-pipe/pull/30
https://github.com/aerogear/aerogear-android-auth/pull/26
https://github.com/aerogear/aerogear-android-authz/pull/26
https://github.com/aerogear/aerogear-android-store/pull/26
https://github.com/aerogear/aerogear-android-security/pull/24

And let me know when you're done.

**Comment by secondsun** [comment]
+1 to what abstrcatjk said

**Comment by danielpassos** [comment]
@abstractj Done! Thanks for the review.

**Comment by abstractj** [comment]
@danielpassos landed

**Pull request merged**